README.rst

```diff
@@ -5,12 +5,13 @@ Mesa: Agent-based modeling in Python 3+
         :target: https://github.com/projectmesa/mesa/actions
 
 .. image:: https://codecov.io/gh/projectmesa/mesa/branch/main/graph/badge.svg
-  :target: https://codecov.io/gh/projectmesa/mesa
+        :target: https://codecov.io/gh/projectmesa/mesa
 
 .. image:: https://img.shields.io/badge/code%20style-black-000000.svg
-    :target: https://github.com/psf/black
+        :target: https://github.com/psf/black
 
-`Mesa`_ is an Apache2 licensed agent-based modeling (or ABM) framework in Python.
+.. image:: https://img.shields.io/matrix/projectmesa:matrix.org?label=Matrix%20chat&logo=Matrix
```

Comment:
I think just labeling it "chat" is simpler. The logo already signifies that it is on Matrix.